### ISSUE TITLE
Custom format newline missing

### ISSUE THREAD

**Issue by aaccomazzi** [adsabs/export_service#132]
It appears we are no longer ouputting a newline at the end of a string constructed with the custom format.  For instance, if I simply specify `%R` as my custom format, all the records' bibcodes are concatenated.  I think we should default to having a newline character at the end of a record.

A user can always modify this default setting using the `%ZEOF` option if necessary.

**Comment by golnazads** [comment]
a user complained that we were putting linefeed at the end when they had not specified, so I removed them. 
Sorry, never implemented EOF when realized not too many people are using it, when looking at classic custom formats, and thought we can use `\n` instead.
https://github.com/adsabs/export_service/issues/97#issuecomment-479085348
Please let me know if %ZEOF should be implemented.

**Comment by aaccomazzi** [comment]
Hmmm... ok, I hate to change things back if this is what people have gotten used to...
But for the record, this doesn't seem to do the trick when I use
```
%ZEncoding:html <LI><A href="%u">%l %Y, "%T," %Q </A>\n
```
I don't see the newline in the output.

**Comment by golnazads** [comment]
Actually it is BBB. export translates it correctly to <br>. For example
```
<LI><A href="https://ui.adsabs.harvard.edu/abs/2019BAAS...51c..89F">Fabbiano, G., Elvis, M., Accomazzi, A., Berriman, G. B., Brickhouse, N., Bose, S., Carrera, D., Chilingarian, I., Civano, F., Czerny, B., D'Abrusco, R., Drake, J., Emami-Meibody, R., Farah, J. R., Fazio, G. G., Feigelson, E., Fornasini, F., Gallagher, J., Grindlay, J., Hernquist, L., Karovska, M., Kim, D.-W., Lacy, G. M., Lazio, J., Maksym, W. P., Martinez Galarza, R., Mazzarella, J., Sanders, D., Scoville, N., Shapiro, I., Siemiginowska, A., Smith, A., Smith, H., Szentgyorgyi, A., Tacchella, S., Thakar, A., Tolls, V., Wilkes, B., Wilner, D., Willner, P., &amp; Wolk, S. J. 2019, "Increasing the Discovery Space in Astrophysics: The Exploration Question for Compact Objects," Astro2020: Decadal Survey on Astronomy and Astrophysics, science white papers, no. 89; Bulletin of the American Astronomical Society, Vol. 51, Issue 3, id. 89 (2019) </A><br / >
```
So what should the behavior be?

**Comment by aaccomazzi** [comment]
Ok, now I do get the same output (must have had a cached export format).  Problem is that export transforms `\n` into `<br / >` so my output of 200 records is one line long!  I would just like to have a manageable output document with newlines at the end of each record, and can't get it.

**Comment by golnazads** [comment]
So EOF is needed. And that does not get encoded right. So for example if we say EOF"\n" then there is a new line. Right?

**Comment by aaccomazzi** [comment]
Sorry, it's `%ZEOL:"string"`, as described here: http://adsabs.harvard.edu/abs_doc/help_pages/customformat.html

But I would also question the choice of turning `\n` into `<br/>` for HTML encoding.  We don't do it for eg. latex (where it would become something like `\\\n`)

**Comment by golnazads** [comment]
Per Alberto, export should only encode the fields coming from solr and leave everything else as is, so basically linefeeds should not be encoded to `<br/>` for html encoding.

**Comment by golnazads** [comment]
implemented in v2.0.68.
